vizro-core/docs/pages/user-guides/selectors.md

```diff
@@ -11,13 +11,13 @@ For instance, the [`Checklist`][vizro.models.Checklist] functions as a multi-opt
 the [`RadioItem`][vizro.models.RadioItems] serves as a single-option selector by default. However, the
 [`Dropdown`][vizro.models.Dropdown] can function as both a multi-option or single-option selector.
 
-For more details, kindly refer to the documentation of the underlying dash components:
+For more details, refer to the documentation of the underlying Dash components:
```

Comment:
Sorry to drag this out, but I have decided that I do not like the presentation here. What bugs me, and I have actively tripped over this, is that these are the docs about our selectors, but the most prominent links are to the underlying components.

What do you think of slightly restructuring this such that the bullets are smth a long those line:

```
For more information, refer to the API reference of the selector, or the documentation of its underlying Dash component:
- [`Checklist`][vizro.models.Checklist] based on [`dcc.Dropdown`](https://dash.plotly.com/dash-core-components/dropdown)
- ...
```